services/marathon/marathon.go

```diff
@@ -158,7 +180,7 @@ func createApps(tasksById map[string][]MarathonTask, marathonApps map[string]Mar
 
 		for _, task := range tasks {
 			if len(task.Ports) > 0 {
-				simpleTasks = append(simpleTasks, Task{Host: task.Host, Port: task.Ports[0]})
+				simpleTasks = append(simpleTasks, Task{Host: task.Host, Port: task.Ports[0], Alive: calculateTaskHealth(task.HealthCheckResults, marathonApps[appId].HealthChecks)})
```

Comment:
There will be delay during because of the event model (chaining from process -> marathon -> bamboo/haproxy). I believe this is also services specific configuration, some services may reduce downtime if healthcheck is configured. 

I hope an optional healthcheck configuration for loadbalancer can be done in Bamboo API. So Bamboo becomes a REST API for the loadbalancer (e.g. HAProxy/Nginx/Apache). Then it's flexible enough to have some setup fit that fits many deployment requirement.

I have't test how long the health will be reflected with the "Alive" flag approach. However I think what you proposed is most simple solution to improve #57